src/patches/vwcBadge.js

```diff
@@ -0,0 +1,21 @@
+/**
+* Patches provided VwcBadge instance to show text in the center
```

Comment:
specify that this is for V3 of badges to not  confuse with v2

Reply:
done